bin/plugin/commands/common.js

```diff
@@ -133,3 +105,57 @@ exports.getModuleData = async ( modulesDir ) => {
 			return firstModule.slug.localeCompare( secondModule.slug );
 		} );
 };
+
+/**
+ * Returns the list of module data for module.
+ *
+ * @param {string} moduleHeader Modules file header contetnt.
+ *
+ * @return {[]WPModuleData} Module data list.
```

Comment:
This doesn't return an array of `WPModuleData`, just a single `WPModuleData`.
```suggestion
 * @return {WPModuleData} Module data.
```